dist/tools/pyterm/pyterm

```diff
@@ -126,9 +128,10 @@ class SerCmd(cmd.Cmd):
         self.json_regs = dict()
         self.init_cmd = []
         self.load_config()
-        self.fmt_str = None
-        if not self.fmt_str:
+        if self.fmt_str == None:
```

Comment:
Since you check above `if formatter` this should lead to an error, if `formatter` is not set (or set to None).